toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/utils/ConfigTesterHelper.java

```diff
@@ -0,0 +1,221 @@
+package io.toast.tk.agent.ui.utils;
+
+import io.toast.tk.agent.ui.NotificationManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.Authenticator;
+import java.net.HttpURLConnection;
+import java.net.InetSocketAddress;
+import java.net.PasswordAuthentication;
+import java.net.Proxy;
+import java.net.SocketTimeoutException;
+import java.net.URL;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLHandshakeException;
+import javax.net.ssl.SSLSession;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * Configuration panel
+ */
+public class ConfigTesterHelper {
+
+	private static final String FILE_DIRECTORY_SEPARATOR = "/";
+	private static final int timeout = 500;
+	
+	public static boolean testWebAppDirectory(String directory, boolean runTryValue, boolean fileOrDirectory)
+			throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)